src/screens/Survey/redux/containers/Container_Question.js

```diff
@@ -0,0 +1,28 @@
+import {connect} from 'react-redux';
+import Question from '../../Question';
+import {writeResponsesToDatabase,loadQuestionResponse,updateTotalQuestions,addKeyToSet,increaseIndex,decreaseIndex,setMaxIndex, SURVEY_NAMESPACE} from '../actions/Actions_question';
+import {getSurveyQuestions} from '../../../../match/redux'
+
+function mapStateToProps(state){
+  return {
+    questionResponses : state[SURVEY_NAMESPACE].questionResponses,
+    index : state[SURVEY_NAMESPACE].index,
+    totalNumQuestions: state[SURVEY_NAMESPACE].totalNumQuestions,
+    questionKeys: state[SURVEY_NAMESPACE].questionKeys,
+    databaseArgs: state[SURVEY_NAMESPACE].databaseArgs,
+    surveyQuestions: getSurveyQuestions(state)
+  }
+}
+
+
+const mapDispatchToProps = dispatch => ({
+  loadQuestionResponse: (questionnum,response) => dispatch(loadQuestionResponse(questionnum,response)),
```

Comment:
I think if all you are doing is calling dispatch on your functions, you can actually use a shortcut syntax where you just have an object with your desired props as keys and your action creators as values. ex: ```{ loadQuestionResponse, increaseIndex, decreaseIndex, setMaxIndex, updateTotalQuestions, writeResponsesToDatabase, addKeyToSet }``` https://github.com/reduxjs/react-redux/blob/master/docs/api.md

Reply:
Yeah, I tried this, but it didn't seem to work cause I had some functions that required parameters. I didn't see any example like that in the api documentation